### PR TITLE
delete frame.row_count in favor of frame.count()

### PR DESCRIPTION
**Pull request by blbarker**

### CODE REVIEW

File: python/sparktk/frame/ops/count.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
 from sparktk.frame.row import Row
 
-def count(self, where):
+def count(self, where=None):
     """
-    Counts qualified rows.
+    Counts all rows or all qualified rows.
```
Comment (blbarker) [inline python/sparktk/frame/ops/count.py:5]: :x: whoops, fix this

Comment (blbarker) [inline python/sparktk/frame/ops/count.py:5]: :o: try now

Comment (blbarker) [inline python/sparktk/frame/ops/count.py:5]: :x: still bad

Comment (blbarker) [inline python/sparktk/frame/ops/count.py:5]: :o:

**Comment by blbarker** [comment]
- [ ] @lewisc 
- [ ] @joyeshmishra 
- [ ] @dmsuehir

File: python/sparktk/frame/ops/count.py (hunk 1 above)
Comment (blbarker) [inline python/sparktk/frame/ops/count.py:5]: :100:  it's good

**Comment by dmsuehir** [comment]
- [x] @dmsuehir  (just kidding)

**Comment by dmsuehir** [comment]
:trollface:

**Pull request merged**